app/src/main/java/com/epam/caloriecalc/data/local/repository/CalorieRepositoryImpl.kt

```diff
@@ -25,6 +25,10 @@ class CalorieRepositoryImpl(
         return dao.deleteIntake(intake)
     }
 
+    override suspend fun getLastIntake(): IntakeRecord {
+      return dao.getLastIntake()
+    }
```

Comment:
I've made the same recommendation for Patrik as well: these one liner non-Unit functions can be shortened to the expression format
`override suspend fun getLastIntake() = dao.getLastIntake()`
Why to scroll so much if the language has these nice features? :) No need to change it now, just keep it in mind

Reply:
That's indeed a nice feature. I will keep this in mind.